client/post-editor/editor-status-label/index.jsx

```diff
@@ -73,7 +81,10 @@ export default React.createClass( {
 				aria-label={ this.translate( 'Show advanced status details' ) }
 				aria-pressed={ !! this.props.advancedStatus }
 			>
-				<Gridicon icon="cog" size={ 16 } />
+				{ this.props.advancedStatus
+					? <Gridicon icon="chevron-up" size={ 18 } />
```

Comment:
It's an downward / upward arrow in the redesign mock-ups. @folletto was this an intended change, or just a copy&paste error or something?

We are talking about this icon:
![sin titulo](https://cloud.githubusercontent.com/assets/1715800/14691719/1a0d6ae0-0753-11e6-81fa-6230f4f93570.png)